services/python_engine/header/rocketjoe/services/python_engine/python_sandbox.hpp

```diff
@@ -30,7 +30,7 @@ namespace rocketjoe { namespace services { namespace python_engine {
         actor_zeta::actor::actor_address address;
         py::scoped_interpreter python_;
         py::module pyrocketjoe;
-        boost::filesystem::path path_script;
+        std::string path_script;
```

Comment:
Why use a std::string ?

Reply:
pybind11 cannot use boost::filesystem::path and crashes with an error.